### ISSUE TITLE
util.io.run_cmd output includes command prompt on Windows

### ISSUE THREAD

**Issue by garret1317** [petzku/Aegisub-Scripts#10]
Output from `util.io.run_cmd` includes the command prompt on Windows, e.g.:
```

C:\Users\dad\AppData\Roaming\Aegisub\log>ffmpeg -i G:\e.mp4 -f rawvideo -pix_fmt rgb24 - 
[actual output starts here]
```
This makes it impossible (or at least unreasonably difficult) to use things from stdout, as there's no way of telling where the actual data starts.

[Test script that demonstrates the problem, and its outputs](https://github.com/petzku/Aegisub-Scripts/files/8794397/petzku.util_windows_out.zip). `ffmpeg_out` is what it should output (and what ffmpeg does if writing to a file), and `petzku_out` is what it actually outputs. You'll probably want to use the small test video so you don't fill your drive if you run it yourself (it outputs raw video).

Would prefer to have it fixed here, and not have to do `if windows this else that`, but if it's not feasible I'll manage.

cc @Zahuczky

**Comment by petzku** [comment]
#11 should have a fix for this. I haven't done comprehensive testing yet, but it seems to work on your test case (after changing the ``io.write`` call to use ``wb`` mode)

**Comment by petzku** [comment]
Should be fixed in #26.
